### PR TITLE
About

### PR DESCRIPTION
**Pull request by charvi-077**

### CODE REVIEW

**Comment by harryjas** [comment]
Squash commits into one.

File: submissions/cm7799/about.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# About Me
+My name is Charvi Mendiratta .I am an ECE student at Panjab University, India.
+# Experience
+I am new to open source.
```
Comment (harryjas) [inline submissions/cm7799/about.md:4]: I am beginner in open source.

File: submissions/cm7799/about.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+# Interests
+electronics coding.
```
Comment (harryjas) [inline submissions/cm7799/about.md]: Capital 'e'

File: submissions/cm7799/about.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+# Contact Info
+Email: [7799cm@gmail.com](mailto :7799cm@gmail.com)
```
Comment (harryjas) [inline submissions/cm7799/about.md]: Wrong format and also add facebook and other social media profiles.

**Comment by kartik-kumarr** [comment]
Given description and check your emoji.

File: submissions/cm7799/about.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+# Emoji
+My favorite emoji is :smile
```
Comment (harryjas) [inline submissions/cm7799/about.md]: change to ':smile:'

**Review by harryjas — CHANGES_REQUESTED** [review]

**Review by harryjas — APPROVED** [review]

**Pull request merged**